src/components/ContentHighlights/data/hooks.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+import {
+  setHighlightStepperModal,
+} from './actions';
+import { initialStepperModalState } from './reducer';
+
+const useStepperModalState = (initialStepperModalStateValue = initialStepperModalState) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [stepperModalState] = useState(setHighlightStepperModal, initialStepperModalStateValue);
```

Comment:
[curious/clarification] I'm not entirely following what's happening here? `useState` only takes 1 argument, the initial state value, but there are 2 arguments to `useState` here.

Also, `stepperModalState` here is an action to manipulate the reducer state, however, I'm not seeing the `stepperModalReducer` actually used anywhere? Is this where a `useReducer` is intended to go?

Reply:
Yep, both assumptions were correct. It was initially a useReducer but I had not renamed it and removed the second parameter.